src/plugins/telemetry_management_section/public/components/opt_in_security_example_flyout.tsx

```diff
@@ -0,0 +1,235 @@
+ * under the License.
+ */
+
+import * as React from 'react';
+
+import {
+  EuiCallOut,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiLoadingSpinner,
+  EuiPortal,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+interface Props {
+  onClose: () => void;
+}
+
+interface State {
+  isLoading: boolean;
+  hasPrivilegeToRead: boolean;
+}
+
+/**
+ * React component for displaying the example security data associated with the Telemetry opt-in banner.
+ */
+export class OptInSecurityExampleFlyout extends React.PureComponent<Props, State> {
+  public readonly state: State = {
+    isLoading: true,
+    hasPrivilegeToRead: false,
+  };
+
+  async componentDidMount() {
+    try {
+      this.setState({
+        isLoading: false,
+        hasPrivilegeToRead: true,
+      });
+    } catch (err) {
+      this.setState({
+        isLoading: false,
+        hasPrivilegeToRead: err.status !== 403,
+      });
+    }
+  }
+
+  renderBody({ isLoading, hasPrivilegeToRead }: State) {
+    if (isLoading) {
+      return (
+        <EuiFlexGroup justifyContent="spaceAround">
+          <EuiFlexItem grow={false}>
+            <EuiLoadingSpinner size="xl" />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      );
+    }
+
+    if (!hasPrivilegeToRead) {
+      return (
+        <EuiCallOut
+          title={
+            <FormattedMessage
+              id="telemetry.callout.errorUnprivilegedUserTitle"
+              defaultMessage="Error displaying cluster statistics"
+            />
+          }
+          color="danger"
+          iconType="cross"
+        >
+          <FormattedMessage
+            id="telemetry.callout.errorUnprivilegedUserDescription"
+            defaultMessage="You do not have access to see unencrypted cluster statistics."
+          />
+        </EuiCallOut>
+      );
+    }
+
+    return (
+      <EuiCodeBlock language="js">
+        {JSON.stringify(this.exampleSecurityPayload, null, 2)}
+      </EuiCodeBlock>
+    );
+  }
+
+  render() {
+    return (
+      <EuiPortal>
+        <EuiFlyout ownFocus onClose={this.props.onClose} maxWidth={true}>
+          <EuiFlyoutHeader>
+            <EuiTitle>
+              <h2>Endpoint security data</h2>
+            </EuiTitle>
+            <EuiTextColor color="subdued">
+              <EuiText>
+                This is a representative sample of the endpoint security alert event that we
+                collect. Endpoint security data is collected only when the Elastic Endpoint is
+                enabled. It includes information about the endpoint configuration and detection
+                events.
+              </EuiText>
+            </EuiTextColor>
+          </EuiFlyoutHeader>
+          <EuiFlyoutBody>{this.renderBody(this.state)}</EuiFlyoutBody>
+        </EuiFlyout>
+      </EuiPortal>
+    );
+  }
+
+  exampleSecurityPayload = {
```

Comment:
It might become a chore to keep the example up to date in the future. Is there a way to pull or poll for an example payload?

Reply:
Things should remain pretty constant, but unfortunately no - The security app would first need to be enabled and a detection would have happened to have occurred. We felt in this senario we would be best just showing a simple sample alert.